homeassistant/components/light/strings.json

```diff
@@ -25,6 +25,65 @@
       "state": {
         "off": "[%key:common::state::off%]",
         "on": "[%key:common::state::on%]"
+      },
+      "state_attributes": {
+        "brightness": {
+          "name": "Brightness"
+        },
+        "color_mode": {
+          "name": "Color mode",
+          "state": {
+            "brightness": "Brightness only",
+            "color_temp": "Color temperature",
+            "hs": "HS",
+            "onoff": "On/Off",
+            "rgb": "RGB",
+            "rgbw": "RGBW",
+            "rgbww": "RGBWW",
+            "unknown": "Unknown",
+            "white": "White",
+            "xy": "XY"
+          }
+        },
+        "color_temp": {
+          "name": "Color temperature in mireds"
+        },
+        "color_temp_kelvin": {
+          "name": "Color temperature in Kelvin"
```

Comment:
Is it necessary to add the units in the name? I assume the units are also shown after the value in the frontend ?

Reply:
> Is it necessary to add the units in the name?

Yes, otherwise you wouldn't understand the difference:

![image](https://user-images.githubusercontent.com/195327/225865216-8af0cfee-09cb-43ae-885f-8950fde8c958.png)

> I assume the units are also shown after the value in the frontend ?

State attributes don't have units.

Reply:
If this state attribute were to be used in a light specific UI, it would be shown with units.

I guess in this generic UI, it would need to have units. Would it be better to replace "in Kelvin" with " (Kelvin)" ?